### PR TITLE
Update maintainer list

### PR DESCRIPTION
**Pull request by dai-chen**
### Description
Following https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md#addition to update maintainer list with nominees. The maintainers have voted and agreed to this nomination and the nominee confirmed their interest in becoming co-maintainer.
 
### Issues Resolved
N/A
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by acarbonetto — APPROVED** [review]

**Review by acarbonetto — APPROVED** [review]

**Comment by dai-chen** [comment]
!!! NOTE !!!: Blocking this PR until we finish the first and second step in https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md#addition

File: sql-cli/README.md
Comment (dai-chen) [reply sql-cli/README.md:4]: This is not relevant but failed the Link Checker action. So fixed it to pass all. :)

**Review by Yury-Fridlyand — APPROVED** [review]

**Review by MaxKsyunz — APPROVED** [review]

**Review by joshuali925 — APPROVED** [review]

**Review by penghuo — APPROVED** [review]

**Comment by dai-chen** [comment]
@dblock As the final step below, do you know how can I find repo admin to change the permission for the new maintainers?

> The repo admin adjusts the new maintainer’s permissions accordingly, and merges the pull request.

**Comment by dblock** [comment]
@dai-chen organizational owners are the only ones that can add external contributors, @CEHENKLE please help?

**Comment by dblock** [comment]
Screen Shot 2022-09-12 at 1 46 03 PM
Screen Shot 2022-09-12 at 1 46 07 PM

**Comment by dai-chen** [comment]
> FYI @anirudha has the same level of access as me here, for next time

Good to know that. Thanks for the help this time!

**Pull request merged**
